runtimes/cloudformation/cmd/handler/main.go

```diff
@@ -57,7 +58,9 @@ func PatchLocalFile(configuration *cfnpatcher.Configuration, ctx context.Context
 		return nil, err
 	}
 
-	result, err := cfnpatcher.Patch(loggerCtx, configuration, inputData)
+	// TODO look for defaults, cannot do better here
```

Comment:
Regarding the TODO - what can we do here? If it's outside the scope of this ticket, can you raise a follow up and link it. I would rather avoiding merging TODO :-)

Reply:
Yes, serverless-patcher is loading the template from a file here. which means that it is running locally and not on a lambda.

>  I would rather avoiding merging TODO :-)

sounds fair 😄 I'll open a follow-up ticket